src/functions/products/createProduct/index.ts

```diff
@@ -11,7 +11,7 @@ export default {
                 path: 'products',
                 cors: true,
                 request: {
-                    schema: {
+                    schemas: {

```

Comment:
Why this change?

Reply:
I fixed two  compiler warnings. 
I changes `schema` in `schemas` because this warning: 
> Serverless: Deprecation warning: Starting with next major version, "http.request.schema" property will be replaced by "http.request.schemas" 

and I split `iamRoleStatements` because this other warning:
> Serverless: Deprecation warning: Starting with version 3.0.0, following property will be replaced: "provider.iamRoleStatements" -> "provider.iam.role.statements"